text/src/autogluon/text/automm/configs/matcher/default.yaml

```diff
@@ -0,0 +1,34 @@
+matcher:
+  match_label: 0
+  matches:
+      - loss:
+          type: "contrastive_loss"
+          pos_margin: 0.8
+          neg_margin: 0.2
+          weight: 1
+        distance:
+          type: "cosine_similarity"
+        miner:
+          type: "pair_margin_miner"
+          pos_margin: 0.6
+          neg_margin: 0.4
+        use_label: True
+        pair:
+          - "Previous Image"
+          - "Current Image"
+#      - loss:
```

Comment:
If this is commented, do we want to remove it? If not, maybe add a TODO saying why we want to leave them here.

Reply:
TODO added.